doc/python/px-arguments.md

```diff
@@ -101,33 +115,60 @@ fig.show()
 Many more examples of wide-form and messy data input can be found in our [detailed wide-form support documentation](/python/wide-form/).
 
 
-### Input Data as Pandas `DataFrame`s
+## Dataframe Input
+
+The first argument of every `px` function is `data_frame`. If you provide a dataframe as a `px` function's first argument, you can then specify column names as strings from the dataframe as other arguments.
+
+### Supported Dataframes
+
+`px` functions natively support the following types of dataframes:
+
+- pandas
+- Polars
+- PyArrow
+- Modin
 
-As shown above, `px` functions supports natively pandas DataFrame. Arguments can either be passed as dataframe columns, or as column names if the `data_frame` argument is provided.
+`px` uses [Narwhals](https://narwhals-dev.github.io/narwhals/) to natively support these dataframes. Other types of dataframes that are currently supported by Narwhals may also work with `px`.
```

Comment:
Move this up above the list to replace the "px functions natively support..." sentence? That feels redundant given this more detailed explanation.

Reply:
Reordered the content